src/functionaltests/java/com/ericsson/ei/utils/HttpRequests.java

```diff
@@ -0,0 +1,146 @@
+        ResponseEntity<String> response = null;
+        String fileContent = "";
+
+        try {
+            fileContent = FileUtils.readFileToString(new File(filePath), "UTF-8");
+        } catch(IOException e) {
+            LOGGER.error(e.getMessage(), e);
+            System.out.println("ERROR: " + e.getMessage());
+        }
+
+        try {
+            jsonParams = new JSONArray(fileContent);
+        } catch(JSONException e) {
+            LOGGER.error(e.getMessage(), e);
+            System.out.println("ERROR: " + e.getMessage());
+        }
+
+        HttpPost httpPost = new HttpPost(url);
+
+        // add request parameters
+        StringEntity params = new StringEntity(jsonParams.toString(), "UTF-8");
+        httpPost.setEntity(params);
+
+        // add headers
+        httpPost.addHeader("content-type", "application/json;charset=UTF-8");
+        httpPost.addHeader("Accept", "application/json");
+
+        response = getResponse(httpPost);
+
+        return response;
+    }
+
+    /**
+     * Delete a given subscription
+     *
+     * The url should include the name of subscription which should be deleted
+     * */
+    public SubscriptionResponse makeHttpDeleteRequest(String url) {
+        ResponseEntity<String> response = null;
+        SubscriptionResponse subscriptionResponse = null;
+
+        HttpDelete httpDelete = new HttpDelete(url);
+        response = getResponse(httpDelete);
+
+        try {
+            subscriptionResponse = mapper.readValue(response.getBody().toString(), SubscriptionResponse.class);
+        } catch(IOException e) {
+            LOGGER.error(e.getMessage(), e);
+        }
+
+        return subscriptionResponse;
+    }
+
+
+    /**
+     * Handle the response from a HTTP request
+     * @param request
+     *      A HTTP request method, e.g. httpGet, httpPost
+     *
+     * @return ResponseEntity
+     *      containing the json content of the http response and statuscode from request
+     * */
+    public ResponseEntity<String> getResponse(HttpRequestBase request) {
+        int statusCode = HttpStatus.PROCESSING.value();
+        String jsonContent = "";
+
+        try(CloseableHttpResponse httpResponse = client.execute(request)) {
+
+            InputStream inStream = httpResponse.getEntity().getContent();
+            BufferedReader bufReader = new BufferedReader(new InputStreamReader(inStream, "UTF-8"));
+
+            for (String line = bufReader.readLine(); line != null; line = bufReader.readLine()) {
+                jsonContent += line;
+            }
+            if (jsonContent.isEmpty()) {
+                jsonContent = "[]";
+            }
+            bufReader.close();
+            inStream.close();
+
+            statusCode = httpResponse.getStatusLine().getStatusCode();
+
+        } catch(IOException e) {
+            LOGGER.error(e.getMessage(), e);
+            System.out.println("ERROR: " + e.getMessage());
```

Comment:
Isn't Logger error visible in the logs. If its visible in the logs,, then you can remove the println printout.